### PR TITLE
Online component fix

### PR DESCRIPTION
**Pull request by goliver5**

### CODE REVIEW

File: StortSpelProjekt/CircularBufferClient.h [hunk 1]
```diff
@@ -2,7 +2,7 @@
 #include<iostream>
 #include<string>
 
-static const unsigned int SIZEOFBUFFER = 20000;
+static const unsigned int SIZEOFBUFFER = 200000;
```
Comment (MorranMogge) [inline StortSpelProjekt/CircularBufferClient.h:5]: Same as first comment I made 🍆

File: Server/CircularBuffer.h [hunk 2]
```diff
@@ -3,7 +3,7 @@
 #include<string>
 #include"PacketsDataTypes.h"
 
-static const unsigned int SIZEOFBUFFER = 2000;
+static const unsigned int SIZEOFBUFFER = 200000;
```
[the remark this answers is not in the document]
Comment (goliver5) [reply Server/CircularBuffer.h:6]: Its ok its only 0,19073 mb so it isnt a problem.

File: Server/Server.cpp [hunk 3]
```diff
@@ -469,16 +496,30 @@ int main()
...
+		//		if(onlineItems[i]->getInUseById() == data.users[j].playerId)
+		//		{
+		//			onlineItems[i]->setPosition(data.users[j].playa.getposition('x'), data.users[j].playa.getposition('y'), data.users[j].playa.getposition('z'));
+		//		}
+		//	}
 		//}
 
+
+		
+		if (((std::chrono::duration<float>)(std::chrono::system_clock::now() - startComponentTimer)).count() > timerComponentLength)
```
[the remark this answers is not in the document]
Comment (goliver5) [reply Server/Server.cpp:512]: probably not but now its possible to do it without bugs.

File: StortSpelProjekt/CircularBufferClient.h (hunk 1 above)
Comment (goliver5) [reply StortSpelProjekt/CircularBufferClient.h:5]: same as first response🤤

**Review by MorranMogge — APPROVED** [review]
Approved （︶^︶）✔

**Pull request merged**